typings/core-js.d.ts

```diff
@@ -0,0 +1,71 @@
+// Type definitions for core-js v0.9.7
+declare type PropertyKey = string | number | symbol;
```

Comment:
Is this file still required, if when the latest version of 'core-js 'and '@types/core-js' is stalled?